### PR TITLE
Add Jasmine testing framework

### PR DESCRIPTION
**Pull request by emilsoman**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+language: node_js
+node_js:
+  - "0.12"
```
Comment (iffyuva) [inline .travis.yml]: is it possible to move to 4.0.0 ?

File: package.json [hunk 2]
```diff
@@ -9,6 +9,13 @@
     "hubot": "2.x"
   },
   "devDependencies": {
-    "hubot": "2.x"
+    "hubot": "2.x",
+    "coffee-script": "1.x"
```
Comment (iffyuva) [inline package.json]: should we use babel + es6?

**Comment by iffyuva** [comment]
PR looks good to me, can be merged.

File: package.json (hunk 2 above)
Comment (emilsoman) [inline package.json]: No, hubot uses coffee. Ideally we don't have to add this in package.json, but I did just to run tests on travis.

File: .travis.yml (hunk 1 above)
Comment (emilsoman) [inline .travis.yml]: @iffyuva I've specified "node" in that list which should use the latest node supported by travis

**Pull request merged**